Basic/Pod/FAQ.pod

```diff
@@ -68,7 +68,7 @@ initiator of the PDL project:
     The PDL concept is to give standard perl5 the ability
     to COMPACTLY store and SPEEDILY manipulate the large
     N-dimensional data sets which are the bread and butter
-    of scientific computing. e.g. $a=$b+$c can add two
+    of scientific computing. e.g. $x=$y+$c can add two
```

Comment:
`$a=$b+$c` is nice.  `$x=$y+$c` is jolting.  Maybe `$x=$y+$z`?

Reply:
I don't disagree, but `$c` has no special meaning in Perl-land, unlike `$a` and `$b`. I didn't want to keep spreading the circles wider and wider, since this is already a 2800+ line change.